### PR TITLE
Added the zero to robot Python example

### PR DESCRIPTION
**Pull request by martinrioux**
Also added a complete code overview, I find it useful to show the big picture.

### CODE REVIEW

File: source/docs/zero-to-robot/step-4/creating-test-drivetrain-program-cpp-java.rst [hunk 1]
```diff
@@ -59,6 +59,33 @@ For C++ projects, there is one more step to set up IntelliSense.  Whenever we op
 .. image:: /docs/software/vscode-overview/images/importing-previous-project/cpp-configurations.png
     :alt: You must choose "Yes" to refresh the C++ configurations.
 
+Basic Drivetrain example
+------------------------
+
+First, here is what a simple code can look like for a Drivetrain with PWM controlled motors (such as SparkMax).
```
Comment (sciencewhiz) [inline source/docs/zero-to-robot/step-4/creating-test-drivetrain-program-cpp-java.rst:65]: Say that for Java/C++ this is the Getting Started example as configured above, and add link for where to get the python getting started example

**Review by virtuald — COMMENTED** [review]
LGTM, except the first line of the file says "Creating your Test Drivetrain Program (C++/Java)", should add python to that?

**Comment by sciencewhiz** [comment]
\inspector check all

**Comment by sciencewhiz** [comment]
\inspector check all

**Review by sciencewhiz — APPROVED** [review]

**Pull request merged**